src/main/java/org/folio/gobi/HelperUtils.java

```diff
@@ -81,8 +81,13 @@ public static String extractIdOfFirst(JsonObject obj, String arrField) {
     return item.getString("id");
   }
 
-  public static String encodeValue(String value) throws UnsupportedEncodingException {
-    return URLEncoder.encode(value, StandardCharsets.UTF_8.toString());
+  public static String encodeValue(String query, Logger logger) {
```

Comment:
If we use `LoggerFactory`, we don't have to pass `Logger` as parameter

Reply:
If HelperUtils has its own logger, it will be difficult to trace back from which class this encode was called. If its passed as a parameter, the caller class name will be present in the logs